src/cbmc/cbmc_parse_options.cpp

```diff
@@ -193,6 +193,9 @@ void cbmc_parse_optionst::get_command_line_options(optionst &options)
   if(cmdline.isset("nondet-static"))
     options.set_option("nondet-static", true);
 
+  if(cmdline.isset("replace-calls"))
```

Comment:
I'm not too convinced that adding this to CBMC is a good idea - it has too many command-line options already.

Reply:
It does feel a bit more like a linker / goto-instrument kind of feature.  It seems like it would be happy living with --remove-function-body and --generate-function-body.